nnvm/tests/python/compiler/test_top_level4.py

```diff
@@ -114,6 +116,22 @@ def test_reduce():
     verify_reduce((4, 4, 3), np.sum, sym.sum, axis=(0, 2))
     verify_reduce((4, 4, 3), np.sum, sym.sum)
 
+    def _with_keepdims(f):
```

Comment:
Just a matter of taste, but I'd suggest that the _with_keepdims definition is before all verify_reduce

```
def test_reduce():
    def _with_keepdims(f):
        ...
    verify_reduce((2, 3, 4), np.max, sym.max, axis=1, keepdims=True)
    ...
```